next.dynamic.mjs

```diff
@@ -41,10 +41,13 @@ const getAllPaths = async () => {
     (locale = '') =>
     (files = []) =>
       sourcePages.map(filename => {
-        const path = filename.replace(nextConstants.MD_EXTENSION_REGEX, '');
-
+        let path = filename.replace(nextConstants.MD_EXTENSION_REGEX, '');
```

Comment:
let's call this pathname and also add an inline comment of what this replace regex does